source/tyr/tyr/api.py

```diff
@@ -88,7 +97,9 @@
 api.add_resource(resources.BssProvider, '/v0/bss_providers', '/v0/bss_providers/<string:id>')
 
 api.add_resource(
-    resources.EquipmentsProvider, '/v0/equipments_providers', '/v0/equipments_providers/<string:id>'
```

Comment:
For endpoint that aren't updated I would define new paths directly to not pollute the resources with this version thing

Reply:
I would say the contrary. For instance, this endpoint isn't affected at all by the version in the resource code